Updated notification/usage.mdx with content-focused edits (#4434)
carbon-design-system/carbon-website: src/pages/components/notification/usage.mdx

> If a toast or inline notification requires a message longer than two lines, use

Reviewer: ```suggestion
If a toast or inline notification requires a message that is longer than two lines, use
```

an actionable notification and include a short message with a “View more” link

> that takes the user to a view of the full notification message. The target of

Reviewer: Could you reword the beginning of this sentence: "The target of..." in the Overflow section? We sometimes refer to "click targets" in our Usage tab documentation and I just don't want to confuse anyone with that terminology. Because target in this case wouldn't be referring to the click target of the link or button.

Author: Done

Reviewer: This still doesn't look like its been updated in this PR. I can help you make the change if you can let me know what you changed the sentence to to remove the word "target".

> that takes the user to a view of the full notification message. The target of

Reviewer: ```suggestion
that takes the user to a view of the full notification message. This destination
```

> the “View more” link can be either a full page with more details or a modal.

Reviewer: ```suggestion
can be either a full page with more details or a modal.
```

### Dismissal

Toast notifications persist by default, but they can timeout and be coded to
dismiss automatically after five seconds on the screen. They can also include a
close button so users can dismiss them sooner. Toast notifications cover content
on the screen so they should always be easily dismissed. Because toast
notifications can dismiss automatically, users should be able to access them
elsewhere after the toast notification disappears if they need more time to read
the notification or who want to refer to it later.

## Actionable

> Actionable notifications are inline or toast notifications that have interactive

Reviewer: > Actionable notifications are inline or toast notifications that have interactive elements.

I would add back somewhere in this sentence that the Actionable notifications are styled as inline or toast notifications, just to be clear that this is a separate variant still.

Author: But we have "Actionable notifications are inline or toast notifications that have interactive
elements." under the Actionable heading already.

elements. Because actionable notifications require user interaction, actionable
notifications come into focus when triggered and, therefore, they disrupt screen

> readers and keyboard users. Only oe action is allowed for each notification, and

Reviewer: ```suggestion
readers and keyboard users. Only one action is allowed for each notification, and
```

Author: Fixed

### Actions

If needed, actionable notifications can include a ghost button for inline

> notifications or a tertiary button for toast notificationss. These action

Reviewer: ```suggestion
notifications or a tertiary button for toast notifications. These action
```

Author: Fixed
